PyRoute/DeltaGalaxy.py

```diff
@@ -0,0 +1,70 @@
+"""
+Created on May 21, 2023
+
+@author: CyberiaResurrection
+"""
+import logging
+import re
+import codecs
+import os
+import ast
+import itertools
+import math
+import networkx as nx
+
+from PyRoute.DeltaDictionary import SectorDictionary, SubsectorDictionary
+from PyRoute.Galaxy import Galaxy, Sector, Subsector, Allegiance
+from PyRoute.Star import Star
+from PyRoute.TradeCalculation import TradeCalculation, NoneCalculation, CommCalculation, XRouteCalculation, \
+    OwnedWorldCalculation
+from PyRoute.StatCalculation import ObjectStatistics
+from PyRoute.AllyGen import AllyGen
+
+
+class DeltaGalaxy(Galaxy):
+
+    def read_sectors(self, sectors, pop_code, ru_calc):
+        sector: SectorDictionary
+
+        sectors.skip_void_subsectors_if_half()
```

Comment:
Avoid wasting time on minimising subsectors that have been emptied out on earlier debugging runs.